src/Time/Time.js

```diff
@@ -163,7 +167,7 @@ class Time extends Component {
                         type={'Period'}
                         upButtonProps={meridiemUpButtonProps}
                         updateTime={this.updateTime}
-                        value={CLOCK[this.state.time.meridiem]} />
+                        value={this.CLOCK[this.state.time.meridiem]} />
```

Comment:
You will need to pass `localizedText` into each `TimeItem` component call since the consumer doesn't have access to that internal component.